### PR TITLE
restructure code

### PR DESCRIPTION
**Pull request by kishandobariy**

### CODE REVIEW

File: myapp/templates/home.html [hunk 1]
```diff
@@ -17,36 +23,162 @@
...
+    <script src="https://rn53themes.net/themes/demo/travelz/js/wow.min.js"></script>
+    <script src="https://rn53themes.net/themes/demo/travelz/js/materialize.min.js"></script>
+    <script src="https://rn53themes.net/themes/demo/travelz/js/custom.js"></script>
+
+
+</body>
+
+
     </html>
-
```
Comment (sahildesai313) [inline myapp/templates/home.html:184]: add new line at the end of file

File: myapp/views.py [hunk 2]
```diff
@@ -1,64 +1,59 @@
...
+            newlname = lastname
+            newphone = phone
+            mymember.firstname = newfname
+            mymember.lastname = newlname
+            mymember.phone = newphone
             mymember.save()
             return redirect('/home')
-        except Register.DoesNotExist:
+        except UserDetail.DoesNotExist:
             print(mymember)
```
Comment (sahildesai313) [inline myapp/views.py:28]: add valid response, that user does not exists

File: myapp/views.py [hunk 3]
```diff
@@ -1,64 +1,59 @@
...
         password = request.POST['password']
-        print("this")
-        print(username, password)
-        user = Register.objects.filter(username=username, password=password)
+        user = UserDetail.objects.filter(username=username, password=password)
         if user.exists():
-          
+
             messages.success(request, 'Successfully Logged In')
             print("login")
```
Comment (sahildesai313) [inline myapp/views.py:47]: remove print function

File: myapp/views.py [hunk 4]
```diff
@@ -89,37 +84,36 @@ def forgot(request):
...
-    return render(request,'register.html')  
+
+            return redirect('/')
+    return render(request, 'register.html')
+
 
 def homepage(request):
-    # user= Register.objects.all().values()
+    # user= UserDetail.objects.all().values()
     # print(user)
```
Comment (sahildesai313) [inline myapp/views.py:117]: if this is not required, remove this commented lines

File: myapp/views.py [hunk 5]
```diff
@@ -128,6 +122,6 @@ def homepage(request):
     print("user")
     print(request.session["username"])
```
Comment (sahildesai313) [inline myapp/views.py:123]: remove print function lines

**Review by sahildesai313 — CHANGES_REQUESTED** [review]

File: myapp/templates/home.html (hunk 1 above)
Comment (kishandobariy) [reply myapp/templates/home.html:184]: Okay will update it

**Comment by kishandobariy** [comment]
@sahildesai313 I updated code as per review, Please approve now

**Review by sahildesai313 — APPROVED** [review]
Looks Good To Me

**Pull request merged**